frontend/src/plugins/router.js

```diff
@@ -14,7 +15,7 @@ export default new Router({
     routes: [
         {
             path: '',
-            redirect: 'login'
+            redirect: 'home'
```

Comment:
Use definition with name, like `redirect: {name: ...}` where name is defined value name on target route